.github/CODEOWNERS

```diff
@@ -2,7 +2,8 @@
 /packages/   @nvie
 /packages/liveblocks-devtools/   @nvie @marcbouchenoire
 /packages/create-liveblocks-app/   @CTNicholas
-/packages/liveblocks-node/   @nvie @ofoucherot @FlowFlorent 
+/packages/liveblocks-node/   @nvie @ofoucherot @FlowFlorent
+/packages/liveblocks-react-comments/   @marcbouchenoire
```

Comment:
I was auto-assigned to review this PR because I'm a code owner for the entire `/packages` directory, but I've opted-out for the `/packages/liveblocks-react-comments` one for now, and made @marcbouchenoire the code owner, so he'll get invited automatically whenever someone touches a file in there going forward (instead of me).